frontend/src/lib/util.ts

```diff
@@ -0,0 +1,8 @@
+export function getFormattedCurrency(n: number) {
+  if (n === 0) return "+ $0";
```

Comment:
Playing with it in a playground, it looks like the code below formats n = 0 as "+ $0" so we can skip this check.

Reply:
Fixed